management/distance-target.yaml

```diff
@@ -0,0 +1,261 @@
+              schema:
+                $ref: './schemas/distance-target.yaml#/schemas/DistanceTarget'
+  '/v1/flight/{flightId}/distance/batch':
+    post:
+      tags:
+        - distance-targeting
+      operationId: batchUpload
+      security:
+        - ApiKeyAuth: []
+      parameters:
+        - name: flightId
+          description: The Flight Id to associate with the Distance Targeting geometry object
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int32
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                ReplaceExisting:
+                  type: boolean
+                  nullable: true
+                Geometries:
+                  type: array
+                  items: 
+                    $ref: './schemas/distance-target.yaml#/schemas/BatchUpload'
+      responses:
+        200:
+          description: The newly uploaded Distance Target(s)
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  InsertedGeometries: 
+                    type: integer
+                    format: int32
+  '/v1/flight/{flightId}/distance/{geometryId}':
+    put:
+      tags:
+        - distance-targeting
+      operationId: update
+      security:
+        - ApiKeyAuth: []
+      parameters:
+        - name: flightId
+          description: The Flight Id to associate with the Distance Targeting geometry object
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int32
+        - name: geometryId
+          description: The Id of the Distance Targeting geometry
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int32
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                Distance:
+                  type: integer
+                  format: float
+                  nullable: true
+                Latitude:
+                  type: integer
+                  format: float
+                  nullable: true
+                Longitude:
+                  type: integer
+                  format: float
+                  nullable: true
+                StreetAddress:
+                  type: string
+                  nullable: true
+      responses:
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: './schemas/distance-target.yaml#/schemas/BadRequest'
+        503:
+          description: Geocoding service unavanilable
+          content:
+            application/json:
+              schema:
+                $ref: './schemas/distance-target.yaml#/schemas/ServiceUnavailable'
```

Comment:
Same here. I'd be surprised if these actually return objects.

Reply:
Updated